src/main/java/create_task_use_case/CreateTaskPresenter.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Create Task Presenter Class.
+ * Author: Fardin Faruk
+ * Modified By: Fardin Faruk
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 27, 2022
+ */
+package create_task_use_case;
+
+public class CreateTaskPresenter implements CreateTaskOutputBoundary {
+    @Override
+    public CreateTaskOutputData prepareSuccessView(CreateTaskOutputData response) {
```

Comment:
Please add javadoc for the methods please.

Reply:
Done.